playground/app.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <div>Nuxt 3 UTM module playground!</div>
-  <pre>{{ $utm }}</pre>
+  <pre>{{ utm }}</pre>
 </template>
 
 <script setup>
-import { useNuxtApp } from "nuxt/app";
+import { useNuxtUTM } from '#imports';
```

Comment:
@hakenprog do you know if auto imports will work?

Reply:
From the Nuxt documentation for modules, I don't think this is possible:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/55b47a66-f348-44d4-9929-1fa1dd0ef8d5">
Link for more [info](https://nuxt.com/docs/guide/going-further/modules)

Reply:
thank you for clarifying this!